### PR TITLE
"Trim mark changed" notification

### LINKED ISSUE #1328: Add notification on trim marker change
Plugins that aggregate over the trim marker range, like e.g. the Offline Surface Tracker when generating a heatmap, need to know when the trim markers change.

### PR DESCRIPTION
**Pull request by papr**
Fixes #1328

### CODE REVIEW

**Comment by papr** [comment]
Ah, correct. I got this error as well, fixed it, but it seems I forgot to push the changes. :(

File: pupil_src/shared_modules/seek_control.py [hunk 1]
```diff
@@ -111,30 +117,62 @@ def current_playback_time(self):
...
+    @property
+    def trim_left(self):
+        return self._trim_left
+
+    @trim_left.setter
+    def trim_left(self, val):
+        self._trim_left = val
+        self.notify_all(
+            {
+                "subject": "seek_control.trim_indeces_changed",
```
Comment (ckbaumann) [inline pupil_src/shared_modules/seek_control.py]: typo: indices or indexes

File: pupil_src/shared_modules/seek_control.py [hunk 2]
```diff
@@ -111,30 +117,62 @@ def current_playback_time(self):
...
+    @property
+    def trim_right(self):
+        return self._trim_right
+
+    @trim_right.setter
+    def trim_right(self, val):
+        self._trim_right = val
+        self.notify_all(
+            {
+                "subject": "seek_control.trim_indeces_changed",
```
Comment (ckbaumann) [inline pupil_src/shared_modules/seek_control.py]: same typo

**Comment by papr** [comment]
Thanks for catching that! Fixed and pushed.

**Pull request merged**